docs/installation.md

```diff
@@ -92,14 +89,17 @@ If you don't want to use this command and set the key yourself, this string shou
 **If the application key is not set, your user sessions and other encrypted data will not be secure!**
 
 #### Additional Configuration
-Lychee needs almost no other configuration out of the box. You are free to get started developing! However, you may wish to review the `config/app.php` file and its documentation. It contains some options that aren't included from `.env` that you may wish to change according to your installation.
 
 You may also want to configure a few additional components of Lychee in `.env`, such as:
```

Comment:
I know that you haven't changed this line, but I wonder why it says "You may also ... such as: ... database ...". Why "may"? The database is something which definitely "must" be configured.

I have to admit that I have never read the install instructions thoroughly, maybe I should have.

Reply:
> The database is something which definitely "must" be configured.

Nope. Composer installs the example into `.env` if it doesn't exist, and the example uses a functional sqlite database.

Reply:
Thanks for clarifying that. I didn't remember that. But then there are some other things which strikes me as wrong.

Some lines below, the guide says

> if you copied the `.env.example` file.

IMHO, this doesn't make sense then, because the file is always copied and set up for the user even if the user does nothing.

Moreover, I still believe that the order in which the information is presented is wrong. The `.env` file should be the primary source of configuration not `config/app.php`. Hence, this should be presented in that order and not the way how it is presented now.

Reply:
> IMHO, this doesn't make sense then, because the file is always copied and set up for the user even if the user does nothing.

It's copied **if** the user does nothing. If it exists, it's not copied. Though I agree that's almost an edge case.

> Moreover, I still believe that the order in which the information is presented is wrong. The `.env` file should be the primary source of configuration not `config/app.php`. Hence, this should be presented in that order and not the way how it is presented now.

I don't understand your objection here - That is how they are presented in this PR, `.env` and then `config/` afterwards?

Reply:
Well, yes after I changed it :grinning: We had a concurrency problem here.

Reply:
That makes sense.